docs/metrics/v3.md

```diff
@@ -20,378 +22,384 @@ Some metrics are bucket specific. These will have a `/bucket` component in their
+| `minio_bucket_replication_proxied_get_requests_failures`            | Number of failures in GET requests proxied to replication target. <br><br>Type: counter                          | `bucket`, `server`                           |
+| `minio_bucket_replication_proxied_get_requests_total`               | Number of GET requests proxied to replication target. <br><br>Type: counter                                      | `bucket`, `server`                           |
+| `minio_bucket_replication_proxied_get_tagging_requests_failures`    | Number of failures in GET tagging requests proxied to replication target. <br><br>Type: counter                  | `bucket`, `server`                           |
+| `minio_bucket_replication_proxied_get_tagging_requests_total`       | Number of GET tagging requests proxied to replication target. <br><br>Type: counter                              | `bucket`, `server`                           |
+| `minio_bucket_replication_proxied_head_requests_failures`           | Number of failures in HEAD requests proxied to replication target. <br><br>Type: counter                         | `bucket`, `server`                           |
+| `minio_bucket_replication_proxied_head_requests_total`              | Number of HEAD requests proxied to replication target. <br><br>Type: counter                                     | `bucket`, `server`                           |
+| `minio_bucket_replication_proxied_put_tagging_requests_failures`    | Number of failures in PUT tagging requests proxied to replication target. <br><br>Type: counter                  | `bucket`, `server`                           |
+| `minio_bucket_replication_proxied_put_tagging_requests_total`       | Number of PUT tagging requests proxied to replication target. <br><br>Type: counter                              | `bucket`, `server`                           |
+| `minio_bucket_replication_sent_bytes`                               | Total number of bytes replicated to the target. <br><br>Type: counter                                            | `bucket`, `server`                           |
+| `minio_bucket_replication_sent_count`                               | Total number of objects replicated to the target. <br><br>Type: counter                                          | `bucket`, `server`                           |
+| `minio_bucket_replication_total_failed_bytes`                       | Total number of bytes failed at least once to replicate since server start. <br><br>Type: counter                | `bucket`, `server`                           |
+| `minio_bucket_replication_total_failed_count`                       | Total number of objects which failed replication since server start. <br><br>Type: counter                       | `bucket`, `server`                           |
```

Comment:
```suggestion
| `minio_bucket_replication_total_failed_count`                       | Total number of objects that failed to replicate since server start. <br><br>Type: counter                       | `bucket`, `server`                           |
```